x-pack/plugins/security_solution/public/common/components/header_global/index.tsx

```diff
@@ -24,13 +24,13 @@ import { APP_ID, ADD_DATA_PATH, APP_DETECTIONS_PATH } from '../../../../common/c
 import { useGlobalHeaderPortal } from '../../hooks/use_global_header_portal';
 import { LinkAnchor } from '../links';
 
-const Wrapper = styled.header`
-  ${({ theme }) => `
+const Wrapper = styled.header<{ $isFixed: boolean }>`
+  ${({ theme, $isFixed }) => `
```

Comment:
It seems like any of the React components could calculate `isFixed`. Why did you choose to calculate it in `HomePageComponent` and then pass it to `Wrapper` via `HeaderGlobal` as opposed to calculating it in `Wrapper` directly?

Reply:
because we need that information also to determine proper `paddingTop` that is being passed to `Main` component (pass `0` if the header is static)